spec/attribute.dd

```diff
@@ -695,20 +695,27 @@ $(P
 $(H3 $(LNAME2 abstract, $(D abstract) Attribute))
 
 $(P
-        If a class is abstract, it cannot be instantiated
-        directly. It can only be instantiated as a base class of
-        another, non-abstract, class.
+        An abstract member function must be overridden by a derived class.
+        Only virtual member functions may be declared abstract; non-virtual 
+        member functions and free-standing functions cannot be declared 
+        abstract.
 )
+
 $(P
-        Classes become abstract if they are defined within an
-        abstract attribute, or if any of the virtual member functions
-        within it are declared as abstract.
+        Classes become abstract if any of the virtual member functions
```

Comment:
if any of **its** member function ~~within it~~ **is** declared declared abstract.